ResearchKit/ActiveTasks/ORKdBHLToneAudiometryStepViewController.m

```diff
@@ -357,8 +354,8 @@ - (void)estimatedBHLAndPlayToneWithFrequency: (NSNumber *)freq {
         self.dBHLToneAudiometryContentView.debugLabel.text = nil;
         [_audioGenerator stop];
     });
-    // adding 0.2 seconds to account for the fadeInDuration which is being set in ORKdBHLToneAudiometryAudioGenerator
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)((preStimulusDelay + toneDuration + 0.2) * NSEC_PER_SEC)), dispatch_get_main_queue(), _pulseDurationWorkBlock);
```

Comment:
previously, `toneDuration` wasn't accounting for the full duration of the stimulus, it now does